docs/nitpick-exceptions

```diff
@@ -85,3 +86,8 @@ py:obj RuntimeError
 py:obj NotImplementedError
 py:obj AttributeError
 py:obj NotImplementedError
+py:obj RendererBase
```

Comment:
I need this change in the release. Should I try to backport the whole PR, or rather just cherry pick this single commit?